src/Stratis.Bitcoin.Features.Wallet.Tests/WalletControllerTest.cs

```diff
@@ -820,7 +820,7 @@ public void GetHistoryWithoutAddressesReturnsEmptyModel()
         {
             string walletName = "myWallet";
             var mockWalletManager = new Mock<IWalletManager>();
-            mockWalletManager.Setup(w => w.GetHistory(walletName, null)).Returns(new List<AccountHistory>
+            mockWalletManager.Setup(w => w.GetHistory(walletName, WalletManager.DefaultAccount)).Returns(new List<AccountHistory>
```

Comment:
Cant we update GetHistory so that the `AccountName` is now an optional parameter?

Reply:
At the moment I'd rather leave it so that only the controllers have the default set. 

If the walletmanager wants to start using it as the default, there's so many places to update it because it isn't treated everywhere else as the default yet.